server/server.go

```diff
@@ -345,6 +348,23 @@ func configureStorage(conf *Config) (storageProvider storage.Store, err error) {
 		}
 	}
 
+	if conf.ChainID == flowgo.Testnet || conf.ChainID == flowgo.Mainnet {
+		// TODO: any reason redis shouldn't work?
```

Comment:
in the current design because the remote impl relies on sqlite instance as the provider, but if refactored I don't think there should be a reason.